### ISSUE TITLE
[0.9.1.8] We need a better UI for bulk item transfers

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1341]
In the late game you might be juggling between a few different large stockpiles or needing to load trucks with a certain specific amount of items that's more than a few stacks but less than "everything" and there isn't a good option for that. Say I have a smelting house and a sawmill house and I need to transport 957 nails and 123 hewn logs there. That means I have to manually count out 9 stacks of 100 nails and then do one partial stack of 57, and then 8 stacks of 15 hewn logs and then 3 more hewn logs. That's a lot of manual counting that ideally you could do in one slider.

It would be easier if say, the stockpiles had some aggregated view that show you how many items of a given type there are and you can move them in bulk precisely, or some other way of doing bulk item transfers.

**Comment by JestinJae** [comment]
I agree that this would be useful, but I suspect it would be very hard to pull off with the current transfer system.  I know you can transfer in bulk by holding the shift key and dragging (or just pressing shift when releasing), but since a stack is an entity of sorts it could be difficult.

However if, when the split panel appeared, it polled the inventory to see how many of the item were there then perhaps it could work?  I haven't looked at the code yet so I'm not certain.

**Comment by ThePiachu** [comment]
If I remember correctly, Dyson Sphere Program solved this issue. AFAIR if you Ctrl Drag it picks up all of the same items from the container into one stack you can deposit somewhere. But if you do Ctrl Shift Drag, when you're trying to deposit items you get a window asking how many items to transfer. That would solve this issue.
